packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantRating/_tests/RestaurantRating.test.js

```diff
@@ -0,0 +1,222 @@
+import { mount } from '@vue/test-utils';
+import RestaurantRating from '../RestaurantRating.vue';
+
+describe('RestaurantRating component', () => {
+    it('should render', () => {
+        // arrange & act
+        const wrapper = mount(RestaurantRating);
+
+        // assert
+        expect(wrapper.exists()).toBe(true);
+    });
+
+    it('renders a summary ratings message for screenreaders with a visually hidden class', () => {
+        // arrange
+        const screenReaderMessage = 'foo';
+
+        const propsData = {
+            mean: 5,
+            count: 250,
+            accessibleMessage: screenReaderMessage
+        };
+
+        // act
+        const wrapper = mount(RestaurantRating, { propsData });
+
+        // assert
+        const screenReaderMessageElement = wrapper.find('[data-test-id="ratings-summary-message"]');
+        expect(screenReaderMessageElement.exists()).toBe(true);
```

Comment:
Does this `const` make sense to sit in the `arrange` section?

Reply:
And same for the others below?

Reply:
It needs to be defined after wrapper is - as creating the wrapper is the 'action' the choice is then to define in `act` or `asset` to my mind it then makes more sense to be defined in arrange, where it's immediately referenced rather than in 'act'

Reply:
Ok cool, I think I'd probably side with `act`  😂  but is there already a convention for this in Fozzie?
If a similar situation exists in other tests  then worth being consistent with that if possible 👍